doc/040_backup.rst

```diff
@@ -50,6 +50,23 @@ still get a nice live status display. Be aware that the live status shows the
 processed files and not the transferred data. Transferred volume might be lower
 (due to de-duplication) or higher.
 
+On windows the ``--use-windows-vss`` option will use windows volume shadow copy
+service (VSS) when creating backups. Restic will transparently create a VSS
+snapshot for each volume that contains files to backup. Files are read from the
+VSS snapshot instead of the regular file. This allows to backup files that are
+exclusively locked by another process during the backup.
+
+By default VSS ignores outlook OST files. This is not a restriction of restic
+but the default windows VSS configuration. The files not to snapshot are
+configured in the windows registry under the following key:
```

Comment:
Please adjust to:

~~~
By default VSS ignores Outlook OST files. This is not a restriction of restic
but the default Windows VSS configuration. The files not to snapshot are
configured in the Windows registry under the following key:
~~~

Reply:
Fixed in https://github.com/restic/restic/pull/2274/commits/3a3a297fe427caa8a2b918b62b4c8a70185d0483